package/gluon-web-network/luasrc/lib/gluon/web/model/admin/network.lua

```diff
@@ -128,7 +128,14 @@ uci:foreach("system", "gpio_switch", function(s)
 		if not section then
 			section = f:section(Section)
 		end
-		local poe = section:option(Flag, s[".name"], translate("Enable " .. s.name))
+
+		local name, port = s.name:match("^(.+[^%d])(%d*)$")
+		local poe
+		if port then
+			poe = section:option(Flag, s[".name"], translatef("Enable " .. name .. " %s", port))
+		else
+			poe = section:option(Flag, s[".name"], translatef("Enable " .. name))
+		end
```

Comment:
Rather than duplicating the whole `section:option(...)`, only the label needs to be set inside the if/else.